tests/everflow/test_everflow_ipv6.py

```diff
@@ -83,9 +83,11 @@ def everflow_direction(self, setup_info):       # noqa F811
 
         yield direction
 
+    @pytest.mark.parametrize("support_kvm_traffic", [False])
```

Comment:
Maybe we can use conditional mark to add a mark to test cases to indicate if traffic test should be skipped on KVM testbed. Then use an auto use fixture here to get the mark and decide whether to skip traffic testing on KVM.

Reply:
Agree, added in latest commit